app/views/onboarding/OnboardingTemplate.tsx

```diff
@@ -47,7 +47,9 @@ const OnboardingTemplate = ({
           backgroundColor='transparent'
           translucent
         />
-        <ImageBackground source={background} style={styles.backgroundImage} />
+        {Layout.screenWidth <= Layout.smallScreenWidth ? null : (
```

Comment:
```suggestion
isLargeScreen = Layout.screenWidth > Layout.smallScreenWidth
...
return (
...
        {isLargeScreen ?  (<ImageBackground source={background} style={styles.backgroundImage} />) : null
```

Reply:
This also strikes me as a bit of a hack, is there a way we can do this without querying the screen width?

Reply:
Maybe introducing @2, @3 size images? I wonder how much runway we can get with that

Reply:
Jacob, would that resize the bg image based on device? Since the copy is going to take up the entire screen for this size phone, and the bg image is mainly texture, so I still think removing the bg for small phones is best. Any suggestions on a less hacky way? I couldn't come up with anything better here.